python_src/src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -291,6 +326,8 @@ def gz_to_pyarrow(
                 for key, value in record.items():
                     table[key].append(value)
 
+        except FileNotFoundError as _:
+            return (None, filename, None)
```

Comment:
this is one of the errors we can hit and we don't need to init the thread when we do.